openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -2101,8 +2101,8 @@ The following is a non-normative example of an object containing the `credential
 The following additional claims are defined for authorization details of type `openid_credential` and this Credential Format.
 
 * `claims`: OPTIONAL. An array of claims description objects as defined in (#claims-description-authorization-details).
-* `credential_definition`: OPTIONAL. Object containing a detailed description of the Credential consisting of the following parameter:
-  * `type`: OPTIONAL. Array as defined in (#server-metadata-jwt-vc-json). This claim contains the type values the Wallet requests authorization for at the Credential Issuer. It MUST be present if the claim `format` is present in the root of the authorization details object. It MUST not be present otherwise.
+* `credential_definition`: REQUIRED. Object containing a detailed description of the Credential consisting of the following parameter:
+  * `type`: REQUIRED. Array as defined in (#server-metadata-jwt-vc-json). This claim contains the type values the Wallet requests authorization for at the Credential Issuer. It MUST be present if the claim `format` is present in the root of the authorization details object. It MUST not be present otherwise.
```

Comment:
```suggestion
  * `type`: REQUIRED if the `format` claim is present with a value of `jwt_vc_json`. It MUST NOT be present otherwise. Array as defined in (#server-metadata-jwt-vc-json). This claim contains the type values the Wallet requests authorization for at the Credential Issuer.
```